Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/Options.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AmazonS3.DownloadObject
```

Comment:
Please use one-line namespace definition. Also, namespace should be `Frends.AmazonS3.DownloadObject`.